apps/app-article/src/components/card.js

```diff
@@ -31,7 +31,13 @@ const Card = (props) => {
                         <div className={"article-premium-badge"}><PremiumBadge/></div>
                         : ''
                 }
-                <img className={"card-img-top"} src={props.article.listImage.url + "&function=hardcrop&width=798&height=649&q=95"}/>
+                {
+                    props.article.listImage.url === null 
+                        ? <img className={"card-img-top"} src={props.article.listImage.url + "&function=hardcrop&width=798&height=649&q=95"}/> 
```

Comment:
This should probably be the `hblDefaultImage`, because `props.article.listImage.url === null` 🙂

Reply:
true, i realized it after I comited :)